src/pages/site/[uuid]/tabs/Overview.tsx

```diff
@@ -29,51 +39,263 @@ interface SiteOverviewTabProps {
 const SiteOverviewTab = ({ site }: SiteOverviewTabProps) => {
   const t = useT();
   const router = useRouter();
-  const { format } = useDate();
+  // const { format } = useDate();
   const { isPPC } = useFramework(site);
+  const [editPolygon, setEditPolygon] = useState(false);
+  const { openModal, closeModal } = useModalContext();
+  const openFormModalHandlerAddPolygon = () => {
+    openModal(
+      <ModalAdd
+        title="Add Polygons"
+        descriptionInput="Drag and drop a GeoJSON, Shapefile, or KML for your site Tannous/Brayton Road."
+        descriptionList={
+          <div className="mt-9 flex">
+            <Text variant="text-12-bold">TerraMatch upload limits:&nbsp;</Text>
```

Comment:
All free text in this component needs to be wrapped in `t()`

Reply:
Done